pages/linux/dockerd.md

```diff
@@ -0,0 +1,20 @@
+# dockerd
+
+> Starting docker daemon, persistent process that manages docker containers.
+> More information: <https://docs.docker.com/engine/reference/commandline/dockerd/>.
+
+- Run docker daemon and config it to listening to specific sockets:
+
+`dockerd --host {{socket1}} --host {{socket2}}`
```

Comment:
I quite not sure how to make this example to be clear. Docker daemon can listen to unix, tcp and also https socket(or port). To name the token as socket1, socket2 is it clear enough?

I need your suggestion please. @sbrl

Reply:
Do you have any suggestion about this one ? @schneiderl